### PR TITLE
Set up the OpenShift labels for bc and dc. #67

### PR DESCRIPTION
**Pull request by WalterMoar**
Set up the buildconfigs and deployconfig for the front end to enable the zero trust networking.

### CODE REVIEW

**Review by bryan-gilbert — APPROVED** [review]

File: ppr-ui/openshift/ppr-ui-bc.yaml [hunk 1]
```diff
@@ -18,9 +20,9 @@ objects:
         FROM bcgov-s2i-caddy
         RUN mkdir -p /var/www/html
         COPY dist /var/www/html
-        # RUN ls -alR /var/www/html
       images:
-      - from:
+      - as: null
+        from:
```
Comment (WalterMoar) [inline ppr-ui/openshift/ppr-ui-bc.yaml:25]: This was preventing the "unchanged" on repeated `oc apply`s.

File: ppr-ui/openshift/ppr-ui-bc.yaml [hunk 2]
```diff
@@ -34,12 +36,13 @@ objects:
           name: bcgov-s2i-caddy:latest
           namespace: openshift
       type: Docker
-      successfulBuildsHistoryLimit: 5
+    successfulBuildsHistoryLimit: 5
```
Comment (WalterMoar) [inline ppr-ui/openshift/ppr-ui-bc.yaml:39]: This was preventing the "unchanged" on repeated `oc apply`s.

File: ppr-ui/openshift/ppr-ui-inter-bc.yaml [hunk 3]
```diff
@@ -29,15 +31,14 @@ objects:
         # ref: branch
       type: Git
     strategy:
-      dockerStrategy:
-        noCache: false
```
Comment (WalterMoar) [inline ppr-ui/openshift/ppr-ui-inter-bc.yaml:33]: This was preventing the "unchanged" on repeated `oc apply`s.

**Review by jguertin — APPROVED** [review]
👍

**Pull request merged**